pyomo/core/expr/numeric_expr.py

```diff
@@ -784,6 +785,45 @@ class MonomialTermExpression(ProductExpression):
     def getname(self, *args, **kwds):
         return 'mon'
 
+class DivisionExpression(ExpressionBase):
```

Comment:
What is the :: notation in the doc string - does this process specially? It would be good to have a note here that indicates args[0] is numerator and args[1] is denominator.

Reply:
It's a typo.

Reply:
I take it back.  That seems to indicate a code block.  It is at least consistent with the other docstrings in that file, and were added as part of the original Pyomo5 expression rewrite.